components/map/map.service.ts

```diff
@@ -6,6 +6,7 @@ import Kinetic from 'ol/Kinetic';
 import Map from 'ol/Map';
 import Static from 'ol/source/ImageStatic';
 import View from 'ol/View';
+import WMTS from 'ol/source/WMTS';
```

Comment:
Move to line 11 as
`import {Cluster, OSM, Vector, WMTS} from 'ol/source';` ?
The same can be done with `ImageStatic as Static`

Reply:
Done